site/content/en/docs/tasks/run_python_jobs.md

```diff
@@ -159,3 +159,120 @@ print(job)
 See the [BatchV1](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/BatchV1Api.md)
 API documentation for more calls.
 
+
+### Flux Operator Job
+
```

Comment:
nit:

can you also add some refereces to and from [Run A Flux MiniCluster](/docs/tasks/run_flux_minicluster/)